fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/final.FLT

```diff
@@ -296,6 +300,40 @@ A32NX_FAC_2_PUSHBUTTON_PRESSED=1
 A32NX_FMS_SWITCHING_KNOB=1
 A32NX_FWC_SKIP_STARTUP=1
 A32NX_GEAR_LEVER_POSITION_REQUEST = 1
+A32NX_PAX_MAIN_FWD_A=35160464
```

Comment:
Can you insert the vars alphabetically? Applies to all flt files

Reply:
will be fixed